crates/re_viewer/src/app.rs

```diff
@@ -971,12 +971,19 @@ impl App {
         while let Some((channel_source, msg)) = self.rx.try_recv() {
             let msg = match msg.payload {
                 re_smart_channel::SmartMessagePayload::Msg(msg) => msg,
+
+                re_smart_channel::SmartMessagePayload::Flush { on_flush_done } => {
+                    on_flush_done();
+                    continue;
+                }
+
                 re_smart_channel::SmartMessagePayload::Quit(err) => {
                     if let Some(err) = err {
                         let log_msg =
                             format!("Data source {} has left unexpectedly: {err}", msg.source);
 
                         #[cfg(not(target_arch = "wasm32"))]
+                        #[cfg(feature = "server")]
```

Comment:
I messed up. There's no feature `server` here. This means that we now do more `warn!` spam on this release than intended.
This happened because the `re_sdk_comms = { workspace = true, features = ["server"] }` fix was above was only done on `main`. When testing there I ran into the same "build re_viewer in isolation" issue again and fixed it incorrectly as seen in the diff here.

Reply:
No worries - this is not a regression. I'm very much looking forward to rustc catching these problems for us…

Reply:
I'll remove it in a follow-up PR so we get a changelog entry for next release

Reply:
Thanks! I re-opened the associated ticket here already https://github.com/rerun-io/rerun/issues/5883